$NetBSD: patch-CMakeLists.txt,v 1.1 2023/11/03 00:06:48 wiz Exp $

Due to some dependencies, we need more than c++0x, but this is last on the command line.

--- CMakeLists.txt.orig	2023-07-04 04:50:29.000000000 +0000
+++ CMakeLists.txt
@@ -31,11 +31,6 @@ if ("${CMAKE_C_COMPILER_ID}" STREQUAL "G
     # compilers (in particular, MSVC 2010 does not support C99)
     set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS} -std=iso9899:1990")
 endif()
-if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU" OR "${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
-elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Intel" AND NOT WIN32)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
-endif()
 # Default to hidden visibility for symbols
 set(CMAKE_C_VISIBILITY_PRESET hidden)
 set(CMAKE_CXX_VISIBILITY_PRESET hidden)
